setup.py

```diff
@@ -80,13 +80,13 @@ def runPackLabextension():
         "Programming Language :: Python :: 3.9",
         "Framework :: Jupyter",
     ],
-    install_requires=["notebook", "nbdime ~=2.0", "packaging", "pexpect"],
+    install_requires=["notebook", "nbdime ~=3.0.0dev0", "packaging", "pexpect"],
```

Comment:
also leaving myself a note this will need to be updated when nbdime is updated as well